src/main/java/org/example/FiltersPage.java

```diff
@@ -39,6 +39,10 @@ public void clearMinPriceField () {
         minPriceField.clear();
     }
 
+    public WebElement getMinPriceField() {
+        return minPriceField;
+    }
+
     public void inputMinPrice (String minPrice) {
         minPriceField.sendKeys(minPrice);
```

Comment:
a custom `sendKeys(WebElement element, String textToType)` method should be written in `waitUtils` class and  used here